selftests/unit/test_utils_iso9660.py

```diff
@@ -0,0 +1,119 @@
+"""
+Verifies the avocado.utils.iso9660 functionality
+"""
+import os
+import shutil
+import sys
+import tempfile
+
+from avocado.utils import iso9660, process
+
+
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+
+class BaseIso9660(unittest.TestCase):
```

Comment:
I actually prefer the way you're calliing the unittests in the b30eccc2f6d880c0ded5bfd792546fbdc27733c5 commit.  Please merge that with this commit.